app/views/generic_files/_browse_everything.html.erb

```diff
@@ -4,6 +4,7 @@
 </div>
 <div class="well">
   <%= form_tag(sufia.generic_files_path, id: 'browse_everything_form', method: 'post') do %>
+      <%= render partial: 'generic_files/upload/to_collection' %>
```

Comment:
Can you make the UI changes dependent on a config value (with a default value of `false` until we're sure this is functionality that the majority of folks want turned on)?